### ISSUE TITLE
Update the edge Version on the store (stuck on 2.17.0 / July 2020)

### ISSUE THREAD

**Issue by tebeco** [reduxjs/redux-devtools#1038]
Hello

I'm seing that this repo latest version is 3.0.5 while the MsEdge store indicate 2.17.0
The latest release there was `Updated July 31, 2020`
So we're almost 2 year without update.

Is there anything special to do in order to get the extensions update ?

**Comment by Methuselah96** [comment]
I submitted 3.0 to Microsoft Edge Add-ons at the same time as the other browsers. Still waiting to be reviewed after 8 days.
![image](https://user-images.githubusercontent.com/693755/150278091-24d6ef35-e3ac-411c-9e3e-718542e57e60.png)

**Comment by tebeco** [comment]
thanks a lot for the super fast answer,
I'm asking that due to the "bug" related to Jira, that hint to update to 3.0.5 ;)

https://jira-software.status.atlassian.com/incidents/5j5754ghwmsl

... because JIRA strikes again
dunno if that submission was 3.0.5 or less ;)

sorry about that, taking the opportunity here

**Comment by tebeco** [comment]
bonus point:
Not sure when the change happen but the old manifest/readme point to the old repo I think:
(that might be fixed already)

> It's an opensource project. See the official repository for more details: https://github.com/zalmoxisus/redux-devtools-extension

**Comment by Methuselah96** [comment]
Yeah, the bug is only present in 3.0.4, so you should be good on any other version above or below that. I'll make sure the description gets updated when 3.0 publishes.

**Comment by tebeco** [comment]
many thx
Have a good day

I'll let you close this one when the Edge Store is up to date I guess ?

**Comment by Methuselah96** [comment]
3.0.2 has been published. Still trying to figure out how to update the description and link. I'll keep this open until I get it in-sync with the latest version in Chrome.

**Comment by Methuselah96** [comment]
3.0.6 is in review with the updated description and link.

**Comment by FaberVitale** [comment]
> 3.0.6 with the updated links has been submitted to the Edge Add-Ons and is in review.

Thanks 👍
